### PR TITLE
add token settings modal & token list

### PR DESCRIPTION
**Pull request by iurimatias**
part 1

### CODE REVIEW

File: src/status/wallet.nim [hunk 1]
```diff
@@ -49,3 +49,6 @@ proc getFiatValue*(self: WalletModel, eth_balance: string, symbol: string, fiat_
   var fiat_balance = parseFloat(eth_balance) * parseFloat(fiat_eth_price)
   echo(fmt"balance in usd: {fiat_balance}")
   fiat_balance
+
+proc hasAsset*(self: WalletModel, account: string, symbol: string): bool =
+  (symbol == "DAI") or (symbol == "OMG")
```
Comment (richard-ramos) [inline src/status/wallet.nim:54]: Why are these assets hardcoded?

File: ui/app/AppLayouts/Wallet/Components/TokenSettingsModalContent.qml [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+import "../../../../shared"
+import "../../Chat/ContactsColumn"
+import "../tokens/"
+
+Item {
+    id: element
+
+    Text {
+        id: modalDialogTitle
+        text: "Add/Remove Tokens"
```
Comment (richard-ramos) [inline ui/app/AppLayouts/Wallet/Components/TokenSettingsModalContent.qml:15]:
```suggestion
        text: qsTr("Add/Remove Tokens")
```

File: ui/app/AppLayouts/Wallet/Components/TokenSettingsModalContent.qml [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+        anchors.top: parent.top
+        anchors.left: parent.left
+        font.bold: true
+        font.pixelSize: 17
+        anchors.leftMargin: 16
```
Comment (richard-ramos) [inline ui/app/AppLayouts/Wallet/Components/TokenSettingsModalContent.qml:20]: I think `Theme.padding` could be used here.

File: ui/app/AppLayouts/Wallet/Components/TokenSettingsModalContent.qml [hunk 4]
```diff
@@ -0,0 +1,151 @@
+import QtQuick 2.3
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.3
+import Qt.labs.platform 1.1
+import "../../../../imports"
+import "../../../../shared"
+import "../../Chat/ContactsColumn"
+import "../tokens/"
+
+Item {
```
Comment (richard-ramos) [inline ui/app/AppLayouts/Wallet/Components/TokenSettingsModalContent.qml:10]: There's a reusable popup dialog that can be used here

File: ui/app/AppLayouts/Wallet/Components/TokenSettingsModalContent.qml [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+                    width: parent.width
+                    height: 40
+                    property bool isVisible: searchBox.searchStr == "" || name.toLowerCase().includes(searchBox.searchStr.toLowerCase()) || symbol.toLowerCase().includes(searchBox.searchStr.toLowerCase())
+                    visible: isVisible ? true : false
+
+                    Image {
+                        id: assetInfoImage
+                        width: 36
+                        height: 36
+                        source: hasIcon ? "../../../img/tokens/" + symbol + ".png" : "../../../img/tokens/0-native.png"
```
Comment (richard-ramos) [inline ui/app/AppLayouts/Wallet/Components/TokenSettingsModalContent.qml:90]: Could xss happen here due to this concatenation?

File: ui/app/AppLayouts/Wallet/Components/TokenSettingsModalContent.qml [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+    Separator {
+        id: footerSeparator
+        anchors.bottom: parent.bottom
+        anchors.bottomMargin: 76
+    }
+
+    StyledButton {
+        anchors.right: parent.right
+        anchors.rightMargin: Theme.padding
+        label: "Apply to all accounts"
```
Comment (richard-ramos) [inline ui/app/AppLayouts/Wallet/Components/TokenSettingsModalContent.qml:136]:
```suggestion
        label: qsTr("Apply to all accounts")
```

File: ui/app/AppLayouts/Wallet/Components/TokenSettingsModalContent.qml [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+        anchors.bottom: parent.bottom
+        anchors.bottomMargin: Theme.padding
+        onClicked: {
+//            console.log(txtCurrency.textField.text)
+//            assetsModel.setDefaultCurrency(txtCurrency.textField.text)
```
Comment (richard-ramos) [inline ui/app/AppLayouts/Wallet/Components/TokenSettingsModalContent.qml:141]:
```suggestion
```

File: src/status/wallet.nim (hunk 1 above)
Comment (iurimatias) [reply src/status/wallet.nim:54]: just for testing purposes (this is "part 1"), other PRs incoming

**Pull request merged**